integration/utils/contLoadGenerator.go

```diff
@@ -45,14 +48,17 @@ type ContLoadGenerator struct {
 	fungibleTokenAddress *flowsdk.Address
 	favContractAddress   *flowsdk.Address
 	accounts             []*flowAccount
-	availableAccounts    chan *flowAccount // queue with accounts that are available for workers
+	availableAccounts    chan         *flowAccount                     // queue with accounts available for   workers
+	happeningAccounts    chan func() (*flowAccount, string, time.Time) // queue with accounts happening after worker processing
```

Comment:
maybe `inUse` instead of `happening` could be more clear?

Reply:
I thought about something like `inUseAccounts` but it seemed a bit vague to me because it didn't seem to differentiate itself enough from the existing `availableAccounts` and I was reluctant to refactor the `availableAccounts` name :-) But I don't mind `happeningAccounts` changing... it seemed at the time like the best of a bad bunch of names :-) Maybe something like `wait2useAccounts` or `wait4blockAccounts`?

Reply:
yeah, maybe something like waitingAccounts